src/Reports.Common/Reports.Common.csproj

```diff
@@ -38,8 +38,8 @@
     </Reference>
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="mscorlib" />
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.12.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
+      <HintPath>..\packages\Newtonsoft.Json.13.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
The path needs to be kept as `$(SolutionDir)` as this project is referenced in another solution as a submodule.

Reply:
Ahh, right.

I suppose every *.report proj needs to have the same `$(SolutionDir)` ref?

Reply:
I'd just leave all of them as `$(SolutionDir)` I think they all end up being referenced in another solution.